cli/src/main/java/io/kestra/cli/commands/tenants/MigrationCommand.java

```diff
@@ -0,0 +1,45 @@
+package io.kestra.cli.commands.tenants;
+
+import io.kestra.cli.AbstractCommand;
+import io.kestra.core.repositories.TenantMigrationInterface;
+import io.micronaut.context.ApplicationContext;
+import jakarta.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+
+@CommandLine.Command(
+    name = "migrate",
+    description = "migrate every elements from the default tenant to a selected tenant"
```

Comment:
```suggestion
    description = "migrate every elements from no tenant to the main tenant"
```

In OSS there is no selected tenant

Reply:
Or maybe there is and on OSS you can only use main?

Reply:
In OSS, you can only use main, you can't choose a tenant. My bad, this is a copy/past